pydantic_extra_types/types/phone_numbers.py

```diff
@@ -2,22 +2,18 @@
 
 from typing import Any, Callable, Generator
 
+import phonenumbers
```

Comment:
This makes phonenumbers a requirement. This is probably fine as it isn't the biggest library, however the ImportError try block no longer makes sense below.

Reply:
Ah thanks it was an issue 👍🏻 

@JamesHutchison 🙏🏻